data.go

```diff
@@ -162,6 +176,10 @@ func ParseOptions(str string) Options {
 		}
 	}
 
+	if options.Quality == 0 {
+		options.Quality = 95
```

Comment:
rather than setting the default here, how about if we do it inside `transformImage` alongside where the size option is converted into its final values.  That way, the options struct always represents exactly what the URL options specified, and also means we can continue to use emptyOptions in the tests.